packages/core/src/util.ts

```diff
@@ -38,6 +38,8 @@ export class Util {
     private static blockTestMap: { [key: string]: { source: string, locator: string, status: string }[]; } = {};
     private static blockTestMapInitialized = false;
 
+    static noOp = (): void => { return };
```

Comment:
why noOp ?

Reply:
otherwise, it gives no statement linting error